### PR TITLE
Consistency fixes

### PR DESCRIPTION
**Pull request by macrologist**
A few small changes, corrections,and bugfixes. 

Some notes:

I'm unsure about the decision I made in commit 15d1558 to prefer `lschedule` and `logical-schedule` over names that end in `scheduler`. It could go either way - I chose this language because "addding to a schedule" makes more intutive sense (to me) than "adding to a schedul**er**".   The commit is meant to address #692 

153e0c9 is meant to address #662. I think I may agree with the now ghostly commenter who suggests that it probably isn't much of an issue. However, disallowing `NIL` as the source of a formal's name while still alowing the name `"nil"` pulled out of a `:NIL` seems like a fine compromise between the two positions. Being wary of `NIL`s with unintended meanings seems like a fine suuperstition to adhere to in Common Lisp.

### CODE REVIEW

**Comment by stylewarning** [comment]
typo in your commit message: "replacement"

File: src/ast.lisp [hunk 1]
```diff
@@ -1329,7 +1329,7 @@ This definition does *not* incorporate the operator description (i.e., any opera
 If this slot is not supplied, then the gate is considered *anonymous*. If this is the case, then the GATE slot must be supplied.")
    ;; N.B. See the generic function GATE-APPLICATION-GATE as well.
    (gate :initarg :gate
-         :initform nil
+         ;:initform nil
```
Comment (stylewarning) [inline src/ast.lisp:1332]: this looks suspect

**Review by stylewarning — CHANGES_REQUESTED** [review]
can we move the LSCHED renaming to a separate PR? I did read it all, and it all looks good barring one `;`, but it'll be easier to read and ensure ok

File: src/ast.lisp (hunk 1 above)
Comment (macrologist) [reply src/ast.lisp:1332]: oh fudge.

**Pull request closed without merging**